client/src/App.js

```diff
@@ -1,22 +1,29 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
 
 function App() {
+  const [data, setData] = React.useState(null);
+
+  /* Fetch API is a tool that manipulates HTTP and, at this moment is to perform requests by using a Get request. To do that
+  is to take in a promise which is a path of /api address that we want to fetch from and make our request. So basically to 
+  fetch information from /api address. */
+  React.useEffect(() => {
+    fetch("/api")
+      .then((res) => res.json())
+      .then((data) => setData(data.message));
+  }, []);
+
+  /* JSX is a syntax extension for JavaScript. It looks like a lot of HTML used for React. Currently, the codes show the curly
```

Comment:
I love this explanation and description! The only thing I might add is on the first line maybe mention it is a templating language, rather than a syntax extension.

```suggestion
  /* JSX is a templating language for React. It looks like a lot of HTML used for React. Currently, the codes show the curly
```